client/app/components/calendar/RoomsContainer.jsx

```diff
@@ -0,0 +1,35 @@
+import React  from 'react'
+
+import './filters.scss'
+
+export default class RoomsContainer extends React.Component {
+    
+    _handleCheckbox(event){
```

Comment:
You could pass the `conferenceRoom` or its `id` directly as argument.
Couple of ways:
1. If you're sticking with the bind in the props then you can just pass `conferenceRoom` as a second argument to bind and it will be passed to the handler
2. If you're binding in the constructor then you can pass an arrow function to `onChange` and call `this._handleCheckbox(conferenceRoom)`
3. If you define another component for the whole filter and pass `conferenceRoom` as prop you can do `2` or define a separate handler that delegates to `props.onToggle` (or whatever you name it) and passes `props.conferenceRoom` as argument